examples/more/RIXS/Ba2YOsO6/Anderson_model/run_rixs_fsolver.py

```diff
@@ -90,8 +90,8 @@
         phi=phi, num_gs=4, nkryl=200, pol_type=poltype_xas, temperature=300
     )
     np.savetxt('xas.dat', np.concatenate((np.array([ominc_xas]).T, xas), axis=1))
-    with open('xas_poles.pkl', 'wb') as f:
-        pickle.dump(xas_poles, f)
+    with open('xas_poles.json', 'w') as f:
+        json.dump(xas_poles, f, indent=4)
```

Comment:
A more compact, but still readable representation can be achieved with `indent=2`.
```suggestion
        json.dump(xas_poles, f, indent=2)
```